### ISSUE TITLE
The People’s Elbow comilla

### ISSUE THREAD

**Issue by antoniablanco** [RodrigoToroIcarte/IIC2113-2023-1#215]
Al estar resolviendo los test case me di cuenta que el titulo de la carta tiene un tipo de comilla "The People’s Elbow" mientras que el que se utiliza en las impresiones de los test case es otro "The People's Elbow". Para corregir esto realice un cambio a mano antes de imprimirlo en pantalla. Quería saber si esto fue un error o a propósito y hay que buscar una solución más limpia. 

PD: Me di cuenta que la diferencia esta solo entre el enunciado y los test, asi que es más limpio de arreglar.
Gracias!

**Comment by RodrigoToroIcarte** [comment]
Sí, las comillas quedan raras en latex. Lo que vale es el título de la carta en el json :)
